src/api/urls.py

```diff
@@ -1,14 +1,12 @@
 from django.urls import include, path
 from rest_framework.routers import DefaultRouter
+from .views import readiness_check, liveness_check
 
 from api import views
 
-router = DefaultRouter()
-router.register(r"users", views.UserViewSet)
-router.register(r"todolists", views.TodoListViewSet)
-router.register(r"todos", views.TodoViewSet)
-
 app_name = "api"
 urlpatterns = [
-    path("", include(router.urls))
+    path('readiness', readiness_check, name='readiness_check'),
+    path('liveness', liveness_check, name='liveness_check'),
+    path('', include(router.urls)),
```

Comment:
The `router` variable is used here, but it is not defined in the file. You need to create an instance of `DefaultRouter` and register your viewsets with it before including its URLs.